api/v1alpha1/bundle_types.go

```diff
@@ -109,10 +109,10 @@ type BundleStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:resource:scope=Cluster
 //+kubebuilder:subresource:status
-//+kubebuilder:printcolumn:name=Reference,type=string,JSONPath=`.spec.source.image.ref`,priority=1
 //+kubebuilder:printcolumn:name=Type,type=string,JSONPath=`.spec.source.type`
 //+kubebuilder:printcolumn:name=Phase,type=string,JSONPath=`.status.phase`
 //+kubebuilder:printcolumn:name=Age,type=date,JSONPath=`.metadata.creationTimestamp`
+//+kubebuilder:printcolumn:name=Resolved Source,type=string,JSONPath=`.status.resolvedSource`,priority=1
```

Comment:
This is hack-ish, but it gets around the problem of there not being a single string field that every source config could fit into to define how that source "resolved".

```
$ k get bundles -o wide
NAME                  TYPE    PHASE      AGE   RESOLVED SOURCE
memcached-api-0.1.0   image   Unpacked   4s    {"image":{"ref":"quay.io/joelanford/memcached-api-bundle@sha256:992ca690771a03dfc552c8f8c0d549f8143f09f3c3d8fbc99e48943b7400415b"},"type":"image"}
memcached-api-0.2.0   image   Unpacked   4s    {"image":{"ref":"quay.io/joelanford/memcached-api-bundle@sha256:7f7de1072ab64df45e9aed826b45b4de46e9c1ed44d40f7e7ab35b645941b8cc"},"type":"image"}
```

Reply:
Can we simply index into the resolvedSource.Ref field here instead of printing out the entire BundleSource object?

Reply:
Not if we also want to support git sources since the path is different.